doc/whats_new/v1.0.rst

```diff
@@ -386,6 +386,10 @@ Changelog
   supporting sparse matrix and raise the appropriate error message.
   :pr:`19879` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| The `fit` method of :class:`preprocessing.OrdinalEncoder` will not
+  raise error when `handle_unknown='ignore'`.
```

Comment:
```suggestion
  raise error when `handle_unknown='ignore'` and unknown categories are given to `fit`.
```

Just trying to be a bit more clear on the changelog. I had to read the code here to understand what the changelog meant.

Reply:
Thank you @adrinjalali  sorry for the confusion